### PR TITLE
Combined duplicate openshift_node_kubelet_args descriptions

### PR DESCRIPTION
**Pull request by ahardin-rh**
Combining  the **openshift_node_kubelet_args** content from these PRs:
- https://github.com/openshift/openshift-docs/pull/1372
- https://github.com/openshift/openshift-docs/pull/1518

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape PTAL. Thanks!

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -134,9 +134,18 @@ link:../../install_config/install/prerequisites.html#managing-docker-container-l
...
 
 |`*openshift_node_kubelet_args*`
-|This variable allows you to specify resources per node using kubelet arguments.
-See link:../admin_guide/manage_nodes.html#configuring-node-resources[Configuring
-Node Resources] for more information.
+|This variable is used to configure `kubeletArguments` on nodes, such as
+arguments used in link:../../admin_guide/garbage_collection.html[container and
+image garbage collection], and to
+link:../../admin_guide/manage_nodes.html#configuring-node-resources[specify
+resources per node] using `kubeletArguments`. `kubeletArguments` are key value
```
Comment (adellape) [inline install_config/install/advanced_install.adoc]: "using kubeletArguments" is redundant here.

File: install_config/install/advanced_install.adoc [hunk 2]
```diff
@@ -134,9 +134,18 @@ link:../../install_config/install/prerequisites.html#managing-docker-container-l
...
+pairs that will be passed directly to the Kubelet that match the
```
Comment (adellape) [inline install_config/install/advanced_install.adoc]: s/will be/are/

File: install_config/install/advanced_install.adoc [hunk 3]
```diff
@@ -134,9 +134,18 @@ link:../../install_config/install/prerequisites.html#managing-docker-container-l
...
+link:http://kubernetes.io/v1.1/docs/admin/kubelet.html[Kubelet's command line
```
Comment (adellape) [inline install_config/install/advanced_install.adoc]: Drop `link:`

**Comment by adellape** [comment]
@ahardin-rh See inline, otherwise LGTM.

**Comment by ahardin-rh** [comment]
Thanks, @adellape ! Content is updated.

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../install_config/install/advanced_install.html[Advanced Installation]

|Combined duplicate`*openshift_node_kubelet_args*` descriptions and moved all of the content to the link:..//install_config/install/advanced_install.html#configuring-host-variables[*Host Variables*] table.
%

**Pull request merged**